### ISSUE TITLE
Version 2.81&2.80 ETH+TON mode:the ton's hashrate lower than lolminer.Plz improve it

### ISSUE THREAD

**Issue by sosme2012** [develsoftware/GMinerRelease#661]
Almost under the same computing power and eth‘s hashrate. The ton's hashrate in lolminer is nearly 1.5 times higher than gminer in my 3080lhr cards

**Comment by CryptoLLC** [comment]
Yes they need to. Ton is slow and almost not worth it on Gminer when mining eth+ton.
